packages/switch/src/mwc-switch.ts

```diff
@@ -0,0 +1,88 @@
+/**
+@license
+Copyright 2018 Google Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+import {FormElement, html, property, observer, query, customElement} from '@material/mwc-base/form-element.js';
+import {style} from './mwc-switch-css.js';
+import MDCSwitchFoundation from '@material/switch/foundation.js';
+
+@customElement('mwc-switch' as any)
+export class Switch extends FormElement {
+  @property({type: Boolean})
+  @observer(function(this: Switch, value: boolean) {
```

Comment:
Another syntax option here is to add a property to the PropertyOptions object (by overriding `createProperty`):

```ts
@property({
  type: Boolean,
  onChange(this: Switch, value: boolean) {
    this.mdcFoundation.setChecked(value);
  },
})
checked = false;
```

Reply:
@sorvell WDYT?

Reply:
Perhaps slightly nicer, but this way it's clearly a separate thing (since it's not part of LitElement).... meh.

Reply:
So is that a no?